presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -547,6 +549,14 @@ public SystemSessionProperties(
                         "Experimental: maximum number of bytes to be collected for dynamic filtering per-driver",
                         featuresConfig.getDynamicFilteringMaxPerDriverSize(),
                         false));
+        queryManagerConfig.getQueryMaxDataSize()
```

Comment:
Could you keep the original `ImmutableList.of(...` so that we can use this session property without setting `query.max-data-size`?

Also, I would recommend moving this to around L250.